modules/tinymce/src/plugins/table/main/ts/actions/ResizeHandler.ts

```diff
@@ -33,12 +33,43 @@ export const getResizeHandler = function (editor: Editor): ResizeHandler {
     return elm.nodeName === 'TABLE';
   };
 
-  const lazyResize = function () {
-    return resize;
-  };
+  const lazyResize = () => resize;
+
+  const lazyWire = () =>
+    wire.getOr(ResizeWire.only(SugarElement.fromDom(editor.getBody())));
+
+  const lazySizing = (table: SugarElement<HTMLTableElement>) =>
+    TableSize.get(editor, table);
+
+  const lazyResizingBehaviour = () =>
+    Settings.isPreserveTableColumnResizing(editor) ? ResizeBehaviour.preserveTable() : ResizeBehaviour.resizeTable();
+
+  const getNumColumns = (table: SugarElement<Element>) =>
+    TableGridSize.getGridSize(table).columns;
+
+  const afterCornerResize = (table: SugarElement<HTMLTableElement>, origin: string, width: number) => {
+    const isRightEdgeResize = Strings.endsWith(origin, 'e');
```

Comment:
As discussed, I tried to do the types for the resize event, but it just becomes very messy due to the dynamic nature. As an example just doing `'bar-' + event.type` or `'corner-' + selectedHandle.name` will never match, since it'll always be interpreted as a string. That means we end up with a heap of casts everywhere that aren't very meaningful as they aren't checking types.

Reply:
I added a comment here though to clarify.